locale/fi/common.json

```diff
@@ -13,8 +13,8 @@
   "footer.description": "Suomi.fi Design System -kirjaston kehityksestä vastaa Väestorekisterikeskus.",
   "footer.links": [
     { "text": "", "url": "" },
-    { "text": "Suomidigi.fi", "url": "https://suomidigi.fi/" },
-    { "text": "", "url": "" }
+    { "text": "Saavutettavuusseloste", "url": "/accessibility-statement" },
```

Comment:
I'm not sure if this needs Gatsby-link but if it works like this it is fine for now. (or even is it just using it actually like navigation etc.)

Reply:
The other footer link was stored here, so the fastest way to implement this was to add the link - no other changes necessary for footer.